packages/components/src/modal/stories/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { boolean, text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../button';
+import Dashicon from '../../dashicon';
+import Modal from '../';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment, useState } from '@wordpress/element';
+
+export default { title: 'Modal', component: Modal };
+
+export const _default = () => {
```

Comment:
This one still needs to be refactored to move hooks out of the story as discussed in other PRs :)

Reply:
@gziolo Ah yes! Thank you. Will get on that :). I think this was an earlier Storybook submission I did, before I discovered the `<Example />` technique.